x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -121,6 +158,45 @@ export const PolicyList = memo(() => {
       defaultMessage: 'Date Created',
     });
 
+    const generateDeployedVersionEntry = (version?: 'latest' | string) => {
```

Comment:
Same here, could we memoize this?

Reply:
This function is inside of a memoized function, since its being used inside of a render props I don't think there is any dependence that could be monitored for change. What do you think?

Reply:
Missed that memoized parent function, looks good to me